doc/uftrace-replay.md

```diff
@@ -48,9 +48,15 @@ REPLAY OPTIONS
 \--libname
 :   Show libname name along with function name.
 
+\--srcline
+:   Show source location of each function if available.
+
 \--format=*TYPE*
 :   Show format style output. Currently, normal and html styles are supported.
 
+-O *LOCATION*, \--loc-filter=*LOCATION*
+:   Set filter to trace selected source locations.
+    This option can be used more than once.  See *FILTERS*.
```

Comment:
Could you add the same description to live, report, graph, dump, script, and tui as well?